CHANGELOG.md

```diff
@@ -10,22 +10,32 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 - Block Teleporter
 - Built in compatability for Better Combat mod
-
+- Cindered ladder and bookshelf
+- Shadewood ladder and bookshelf
+- Parrots will mimic imps, ogres, basilisks, and liches
+- 
 ### Changed
 - Structure Block Registry Processor can now be used for data generated structures through a processor_list
 - Adjusted the hitbox size for Lotus Flowers
 - The Lotus Time Capsule now spawns a Lotus Flower when placed
 - Lotus Time Capsule blocks now remove any Lotus Flowers above them when destroyed
+- Cruxite button now uses raw cruxite instead of cruxite block
+- Many organic blocks are now compostable
+- Many wood-based blocks can now be used as fuel
+- Cruxite ore textures tweaked
 
 ### Fixed
 - The Umbral Infiltrator no longer jitters when thrown by a void player
 - Manually-spawned Lotus Flowers can no longer be hurt or killed by any damage source
 - Lotus Flowers no longer push entities inside of them away
 - Ogres and Basilisks no longer slide around when idling
+- Cruxite block now functions properly as a storage block
+- Uranium block now functions properly as a storage block
```

Comment:
This is in reference to the button recipe change, right?
This entry should probably be enrolled into the cruxite button recipe change then, or alternatively removed outright:
```suggestion
```

Reply:
Alright. I broke it up that way because it was one fix, but two distinct reported issues ("storage blocks don't work" and "the recipe for buttons is too expensive"). So I wanted to keep it clear to end users who might not really understand how recipes work but it can be removed. It's not critical or anything.

Reply:
That is reasonable. If you want to add back a changelog entry for the cruxite "storage block fix", I'd be fine with it as long as it is rewritten to be more specifically about the recipe, something like "fixes recipe conflict for recipe turning cruxite block back into raw cruxite".

Reply:
Updated.